kv/union_store.go

```diff
@@ -127,7 +127,7 @@ func (us *UnionStore) CheckLazyConditionPairs() error {
 	for it := us.lazyConditionPairs.NewIterator(nil); it.Valid(); it.Next() {
 		if len(it.Value()) == 0 {
 			if _, exist := values[it.Key()]; exist {
-				return errors.Trace(ErrKeyExists)
```

Comment:
I need a formatted string to indicate which duplicate key, not a fixed error.

can I use：`errors.Errorf("Error: key already exist in %s",it.Key())`

errors is from package:"github.com/juju/errors"